packages/centreon-ui/src/Submenu/SubmenuHeader/index.tsx

```diff
@@ -13,6 +15,77 @@ import IconToggleSubmenu from '../../Icon/IconToggleSubmenu';
 import SubmenuItems from './SubmenuItems';
 import SubmenuItem from './SubmenuItem';
 
+interface StyleProps {
+  CounterRightTranslation?: number;
```

Comment:
variables are named using camelCase

```suggestion
  counterRightTranslation?: number;
```